extensions/ql-vscode/src/view/method-modeling/MethodModeling.tsx

```diff
@@ -1,9 +1,35 @@
 import * as React from "react";
+import { styled } from "styled-components";
+import { ModelingStatusIndicator } from "../model-editor/ModelingStatusIndicator";
+
+const Container = styled.div`
+  background-color: var(--vscode-peekViewResult-background);
+  padding: 0.3rem;
+  margin-bottom: 1rem;
+`;
+
+const Title = styled.div`
+  padding-bottom: 0.3rem;
+  font-size: 1.2em;
+`;
+
+const DependencyBox = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const DependencyName = styled.span`
+  font-family: var(--vscode-editor-font-family);
+`;
 
 export const MethodModeling = (): JSX.Element => {
   return (
-    <>
-      <p>Hello</p>
-    </>
+    <Container>
+      <Title>API or Method</Title>
+      <DependencyBox>
+        <DependencyName>that.dependency.THENAME</DependencyName>
```

Comment:
What do you think about receiving this, and the status as a prop? Fine if you want to do it in a separate PR, just wondered because of the three stories.

Reply:
Discussed offline, we'll leave as is an deal in a separate PR.

Reply:
Sounds good!